test-alerts.cpp

```diff
@@ -5,4 +5,70 @@
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+  REQUIRE(inferBreach(35, 20, 30) == TOO_HIGH);
+  REQUIRE(inferBreach(25, 20, 30) == NORMAL);
+  REQUIRE(inferBreach(20, 20, 30) == NORMAL);
+  REQUIRE(inferBreach(30, 20, 30) == NORMAL);
+}
+
+TEST_CASE("gets breach values based on cooling type")
+{
+  //Get HIGH and LOW breach values.
+  std::pair<int, int> breachValues = getTemperatureBreachValues(PASSIVE_COOLING);
+  REQUIRE(breachValues.first == 0);
+  REQUIRE(breachValues.second == 35);
+  breachValues = getTemperatureBreachValues(HI_ACTIVE_COOLING);
+  REQUIRE(breachValues.first == 0);
+  REQUIRE(breachValues.second == 45);
+  breachValues = getTemperatureBreachValues(MED_ACTIVE_COOLING);
+  REQUIRE(breachValues.first == 0);
+  REQUIRE(breachValues.second == 40);
+}
+
+TEST_CASE("classifies Temperature Breach") {
+  REQUIRE(classifyTemperatureBreach(PASSIVE_COOLING,5) == NORMAL);
```

Comment:
How is the boundary values handled - 0 is considered as normal or too low, 35 is considered as normal or too high?
You are free to assume and tests should reflect these assumptions.
Such conditions should be clear from your tests. Ambiguity may lead to errors later.